sound/soc/sof/sof-client.c

```diff
@@ -221,6 +221,52 @@ int sof_client_ipc_tx_message(struct sof_client_dev *cdev, void *ipc_msg,
 }
 EXPORT_SYMBOL_NS_GPL(sof_client_ipc_tx_message, SND_SOC_SOF_CLIENT);
 
+int sof_suspend_clients(struct snd_sof_dev *sdev, pm_message_t state)
```

Comment:
It seems strange to me that these are needed, that seems like something that the generic PM should be handling.